README.md

```diff
@@ -23,6 +23,8 @@ If you're not able to run through this guide yourself yet, watch one of us go th
 
 [![Watch the video](https://img.youtube.com/vi/GQZ1Wcy1cQM/maxresdefault.jpg)](https://youtu.be/GQZ1Wcy1cQM?t=914)
 
+> *Note*: There have been some UI and flow changes to XTP since this video was recorded. Use this README as the definitive guide.
```

Comment:
I think i may fast follow by replacing this video with a new, more focused walkthrough of me doing the guide.